### ISSUE TITLE
hackishlyHidesInputAccessoryView not found on object of type UIView

### ISSUE THREAD

**Issue by ptarjan** [driftyco/ionic-plugin-keyboard#135]
I'm getting this using XCode 7.0.1

![image](https://cloud.githubusercontent.com/assets/40143/10260879/e4ad0c00-6934-11e5-98e7-5b229f829bd2.png)

**Comment by ptarjan** [comment]
Nevermind, I was using com.ionic.keyboard

**Comment by ptarjan** [comment]
And still present in the new version

**Comment by RangerRick** [comment]
Yup.  Looks like ionic-plugin-keyboard has not been updated for cordova-ios-4 yet.

**Comment by raymatos** [comment]
Any update on this?  workaround?

**Comment by macdonst** [comment]
Update to version 1.0.8 as it is fixed.
